robot/pmm/tests/browser/ProgramCohort/Create_Program_Cohort.robot

```diff
@@ -15,29 +15,73 @@ Setup Test Data
-     Click Wrapper Related List Button      Program Cohorts                         New
+     Go To Page                             Listing                 ${ns}ProgramCohort__c   
+     Click Object Button                    New
+     wait for modal                         New                     Program Cohort
+     populate field                         Program Cohort          ${program_cohort}  
+     select value from dropdown             Status                  Planned
+     populate lookup field                  Program                 ${program}[Name]
+     populate field                         Description             New Program Cohort via Robot Test
+     select date from datepicker            Start Date              10
+     select date from datepicker            End Date                25
+     Click modal button                     Save
+     Wait Until Modal Is Closed
+     verify page header                     Program Cohort
+     verify details                         Program Cohort          contains        ${program_cohort}
+     verify details                         Status                  contains        Planned
+     verify details                         Program                 contains        ${program}[Name]
+     verify page contains related list      Program Engagements
+     verify page contains related list      Files
+     ${program_cohort_id} =                 Save Current Record ID For Deletion     ${ns}ProgramCohort__c
+
+
+Date validation when Start date is later than end date
+     [Documentation]                        This test opens the new program cohort dialog and enters a end date earlier than start date
+     ...                                    and verifies that an error message is displayed
+     [tags]                                 W-037572  feature:Program Cohort
+     Go To Page                             Listing                 ${ns}ProgramCohort__c   
+     Click Object Button                    New
+     wait for modal                         New                     Program Cohort
+     populate field                         Program Cohort          ${program_cohort}  
+     select value from dropdown             Status                  Planned
+     populate lookup field                  Program                 ${program}[Name]
+     populate field                         Description             New Program Cohort via Robot Test
+     select date from datepicker            Start Date              25
+     select date from datepicker            End Date                10
+     Click modal button                     Save
+     verify modal error                     Start Date must be before End Date
+
+Date validation
+    [Documentation]                         This test opens the new program dialog and enters a end date earlier than start date
+    ...                                     and verifies that an error message is displayed
```

Comment:
can u modify this documentation to match what exactly being done like open program edit dates and save. create program coharts from related list so on

Reply:
updated the documentation